packages/docs/src/components/landing/LandingEpicmaxBanner.vue

```diff
@@ -50,10 +50,7 @@ export default class EpicmaxBanner extends Vue {
       display: flex;
```

Comment:
I suggest we compress images a bit, 240/240 might be enough

<img width="673" alt="image" src="https://user-images.githubusercontent.com/5394573/194244732-ef110573-7a1d-4c54-9834-a29da7c1336d.png">